public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,16 @@
+# Python API
+
+DVC can be used as a Python library, simply [install](/doc/install) it with a
+package manager like `pip` or `conda`, and as a Python
+[project requirement](https://pip.pypa.io/en/latest/user_guide/#requirements-files)
+if needed. This reference provides the details about the functions in the API
+module `dvc.api`, which can be imported any regular way, for example:
+
+```py
+import dvc.api
+```
+
+The purpose of our API is to provide programatic access to the data or models
+[stored and versioned using DVC](/doc/use-cases/versioning-data-and-model-files)
```

Comment:
probably it can access any data, including regular Git

Reply:
I'm not sure, can you maybe confirm @Suor? Can `open()` get a regular Git-tracked file? Thanks

Reply:
Git-tracked files are not supported according to my tests with the latest DVC version. I get

`Unable to find DVC-file with output 'path/to/git/tracked/file'`

Reply:
^ tested with `get_url(()`

Reply:
we should test/see the code for open/read as well. get-url is very specific function. It's like show-url in get that does not support Git-tracked files either.

Reply:
Ahhh, `read()` does work. So `open()` also must work. Just `get_url` doesn't support Git-tracked files. Which makes sense... Will update ⏳

Reply:
Updated in 1d5d3a9.